github/repos_pages.go

```diff
@@ -36,6 +44,46 @@ type PagesBuild struct {
 	UpdatedAt *Timestamp  `json:"updated_at,omitempty"`
 }
 
+// EnablePageSite allows to enable GitHub Pages
+//
+// https://developer.github.com/v3/repos/pages/#enable-a-pages-site
```

Comment:
Let's please be consistent with the rest of this repo and change this to:

```go
// GitHub API docs: https://developer.github.com/v3/repos/pages/#enable-a-pages-site
```

Reply:
Thanks, I will keep it in mind from future